src/mpid/ch3/src/mpid_vc.c

```diff
@@ -331,7 +331,7 @@ int MPID_GPID_GetAllInComm( MPIR_Comm *comm_ptr, int local_size,
 	}
 	*gpid++ = vc->pg_rank;
 
-        MPL_DBG_MSG_FMT(MPIR_DBG_COMM,VERBOSE, (MPL_DBG_FDEST,
+        MPL_DBG_MSG_FMT(MPIDI_CH3_DBG_OTHER,VERBOSE, (MPL_DBG_FDEST,
```

Comment:
Probably put this in separate commit, as it's not related to the functionality changes.